### PR TITLE
fix(ontology-editor): bring back the gui-attr for pulldown (DEV-856)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DEV-856

### CODE REVIEW

File: src/app/project/ontology/property-form/property-form.component.ts [hunk 1]
```diff
@@ -337,31 +337,35 @@ export class PropertyFormComponent implements OnInit {
...
-                    this.showGuiAttr = false;
+            if (type.objectType) {
+                switch (type.objectType) {
+                    // prop type is a list
+                    case Constants.ListValue:
+                        this.showGuiAttr = true;
+                        // gui attribute value for lists looks as follow: hlist=<http://rdfh.ch/lists/00FF/73d0ec0302>
+                        // get index from guiAttr array where value starts with hlist=
+                        const i = this.guiAttributes.findIndex(element => element.includes('hlist'));
+                        // find content beteween pointy brackets to get list iri
```
Comment (mdelez) [inline src/app/project/ontology/property-form/property-form.component.ts:348]: beteween -> between

File: src/app/project/ontology/property-form/property-form.component.ts [hunk 2]
```diff
@@ -337,31 +337,35 @@ export class PropertyFormComponent implements OnInit {
...
-                    break;
-
-                default:
-                    this.showGuiAttr = false;
+            if (type.objectType) {
+                switch (type.objectType) {
+                    // prop type is a list
+                    case Constants.ListValue:
+                        this.showGuiAttr = true;
+                        // gui attribute value for lists looks as follow: hlist=<http://rdfh.ch/lists/00FF/73d0ec0302>
```
Comment (mdelez) [inline src/app/project/ontology/property-form/property-form.component.ts:345]: follow -> follows

Comment (kilchenmann) [reply src/app/project/ontology/property-form/property-form.component.ts]: resolved in c9708892

File: src/app/project/ontology/property-form/property-form.component.ts (hunk 1 above)
Comment (kilchenmann) [reply src/app/project/ontology/property-form/property-form.component.ts]: resolved in c9708892

**Pull request merged**